build.cake

```diff
@@ -36,6 +36,7 @@ var GUI_TESTS = BIN_DIR + "nunit-gui.tests.dll";
 // Package sources for nuget restore
 var PACKAGE_SOURCE = new string[]
 {
+    "https://www.nuget.org/api/v2",
```

Comment:
Ah! it's also because VS doesn't have any rules for the layout of .cake files. I wonder if we can add a set.